src/components/Prescription/Prescription.css

```diff
@@ -1,12 +1,21 @@
 .script-container {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    margin-top: 22vh;
     border: black solid 3px;
     background-color: white;
-    height: 88%;
-    width: 95%; 
+    height: 70vh;
+    width: 40vw; 
     border-radius: 10px;
+}
+
+.top-container {
     display: flex;
-    align-items: flex-start;
-    flex-wrap: wrap;
```

Comment:
This seemed to rely on the wrap to control the layout, when I removed it the layout broke. I refactored the component to not rely on this. I think it is a sturdier layout.